SetupDataPkg/Include/Library/ConfigSystemModeLib.h

```diff
@@ -0,0 +1,26 @@
+/** @file
+  Library interface for system mode related functions for configuration modules.
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef CONFIG_SYSTEM_MODE_LIB_H_
+#define CONFIG_SYSTEM_MODE_LIB_H_
+
+/**
+  This routine indicates if the system is in Manufacturing Mode.
+
+  @retval  ManufacturingMode - Platforms may have a manufacturing mode.
+                               DFCI Auto opt-in's the management cert included
+                               in the firmware volume in Manufacturing Mode.

```

Comment:
Is this DFCI comment relevant for this?

Reply:
Incorrect comments creeped in. Will clear it.

Reply:
Updated.